### ISSUE TITLE
Addition to README

### ISSUE THREAD

**Issue by Dextrome** [whalehub/noita-map-viewer#4]
I noticed just placing the iiif3 folder of your own noita map in the noita-map-viewer folder isn't enough

You also have to host the folder on a http fileserver, and set the "id" in .\noita-map-viewer\iiif3\info.json to the URL of that fileserver

Maybe this is common knowledge but it took me a while to figure it out :)

**Comment by whalehub** [comment]
The webserver requirement is mentioned in the README, but you're correct that it doesn't mention the need to set the `id` in that file. I'll edit the README to reflect that requirement - thanks!

**Comment by aremic-caylent** [comment]
I ran into this issue right now. 
I'm hosting it locally using webby and I tried different values for the id field, but all I get is a black map with the correct location of the places of interest. 
I tried setting the id to "http://localhost:8000/", "http://localhost:8000/index.html", http://localhost:8000/iiif3", etc, but I always end up with the black map. 

Maybe an examples folder would prevent other people from running into this issue as well.
